agent/tcs/handler/types.go

```diff
@@ -50,3 +51,10 @@ func (params *TelemetrySessionParams) time() ttime.Time {
 	})
 	return params._time
 }
+
+func (params *TelemetrySessionParams) isMetricsDisabled() (bool, error) {
+	if params.Cfg != nil {
+		return params.Cfg.DisableMetrics && params.Cfg.DisableDockerHealthCheck, nil
```

Comment:
Just for understanding, is `DisableMetrics` something that we have introduced previously but never used? Would it be better to add a description in the PR summary for it?

Reply:
`DisableMetrics` maps to `ECS_DISABLE_METRICS` and is already covered in the README. this PR is to add a toggle for the health checks, which relies on the underlying tcs/metrics subsystem.